hacs.json

```diff
@@ -1,4 +1,4 @@
 {
-  "name": "OSM Geocode",
+  "name": "OSM Geocode"
```

Comment:
Why?

Reply:
Just discovered nominatim and quickly found this addon.
I'd like to add it via hacs, but it complains  with

`trailing comma is not allowed: line 7 column 33 (char 197)`

I assume this PR fixes this.

Please merge!

Thank you both for the addon and the fix!